src/trackers/helpers/parseUrl.js

```diff
@@ -1,51 +0,0 @@
-// parse a URL using a few different packages since none give us all the info that we want
-const tldjs = require('tldjs')
-const tldts = require('tldts')
-const urlParse = require('url')
-
-class URL {
-    constructor (url) {
-        url = url.replace(/^blob:/, '')
```

Comment:
We had two weird parts in this old code - this `blob:` thing above and appending `http://` if url fails to parse (line 40 bellow). As far as I can see we are not moving those over to the new code and I wonder if this will have some unintended consequences. Do you think that we need to port those over @jdorweiler ?

Reply:
The old parser didn't like `blob:`, so the easiest thing was to remove it and parse as a normal URL. @sammacbeth how does the new one handle these?

Appending `http` was added for CNAMES, @Charlie-belmer would know more about that.

Reply:
True - there are some urls reported as `blob:` URLs. This seems strange, maybe a bug in the collector? I'll add the same fix to this code to simply ignore the blob prefix.

Reply:
I think those blobs are web workers - I can double check if you have an example of a site where this happens on hand.

Reply:
@kdzwinel `https://1.1.1.1/pt-BR/dns/` has a request for `blob:https://1.1.1.1/fb597db6-9b8d-4344-90d0-a753765f16a8` (in crawl file `/mnt/efs/crawler-data/2020-12-14/US/3p-crawl/1.1.1.1_5975.json`)

Reply:
Yeah, it's a webworker. Looks like a test if webworkers are available so that the third party video service knows if it can use them.
<img width="1383" alt="cf-blob" src="https://user-images.githubusercontent.com/985504/107057299-975ac200-67d3-11eb-8f62-6c951f0f1bd6.png">